### PR TITLE
Features/homepage

### PR DESCRIPTION
**Pull request by AhmedMohsen600**
In this branch we:
 
       - Added Linters for HTML&CSS
       - Added lighthouse library
       - Created index.html file and added h1 tag with a class of title
       - Created style.css and change the color of the title
       - Added test.md to .gitignore file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,5 +9,6 @@
 </head>
 <body>
      <h1 class="title">Hello Microverse!</h1>
+     <div>misssed the pull reuqest link and doing recommit again</div>
```
Comment (Qoosim) [inline index.html]: - [x] Kindly remove this line, as it is not part of the requirements.

**Review by Qoosim — CHANGES_REQUESTED** [review]
# *Status: Required Changes* :recycle:

Hi @AhmedMohsen600, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  Correct GitHub Flow. :heavy_check_mark: 
-   Meaningful commit messages. :heavy_check_mark:




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [x] Kindly check your linter setup and make it run the tests on your pull request. As at now, the linter tests are not running on your repo.

- [x] Kindly update your README file with your information as to make it look professional. You can check on [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) for insight on how to make a professional README.

**Comment by AhmedMohsen600** [comment]
@Qoosim How I can run the linters tests?

**Comment by Qoosim** [comment]
Have you setup the linters by following the instructions the project requirements?

**Comment by AhmedMohsen600** [comment]
@Qoosim  Yes I have done everything

**Comment by Qoosim** [comment]
Check your slack

**Review by mckent05 — APPROVED** [review]
Hi @AhmedMohsen600 , 

# STATUS: APPROVED 🎉 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**